modules/charon-core/src/main/java/org/wso2/charon3/core/objects/Group.java

```diff
@@ -342,4 +342,72 @@ public void setSchemas() {
         return memberIds;
     }
 
+    /**
+     * Set the associated roles of the group.
+     *
+     * @param role Role object.
+     * @throws CharonException     CharonException.
+     * @throws BadRequestException BadRequestException.
+     */
+    public void setRole(Role role) throws CharonException, BadRequestException {
+
+        SimpleAttribute valueSimpleAttribute = null;
+        SimpleAttribute displaySimpleAttribute;
+        SimpleAttribute referenceSimpleAttribute;
+        String reference = role.getLocation();
+        String value = role.getId();
+        String display = role.getDisplayName();
+        ComplexAttribute complexAttribute = new ComplexAttribute();
+
+        if (value != null) {
```

Comment:
Can't  we use the `StringUtils.isNotEmpty` for `value` not null check. Since value may be "' as well. If so we can change in other places as well.

Reply:
Added the StringUtils to the repo.